http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -416,4 +484,112 @@ private static boolean hasScope(String scopeParam, String targetScope) {
         }
         return false;
     }
+    private String convertToRequestParameter(URIBuilder redirectUriBuilder, String redirectUri, String state, List<NameValuePair> forwardedQueryParams) throws Exception {
+        redirectUriBuilder.addParameter(SCOPE, OIDC_SCOPE);
+        forwardedQueryParams.add(new BasicNameValuePair(STATE, state));
```

Comment:
Do we need to update the forwarded query params?

Reply:
changed it to add them as JWT claims directly instead of adding it to forwarded query params first